packages/core/src/util/spanFilter.js

```diff
@@ -143,7 +143,7 @@ function shouldIgnore(span, ignoreEndpointsConfig) {
     // For e.g., for a Kafka, the ignoreconfig like { methods: ['consume'], endpoints: ['t1'] }.
     if (typeof ignoreconfig === 'object') {
       // Case where ignoreconfig does not specify any filtering criteria.
-      if (!ignoreconfig.methods && !ignoreconfig.endpoints) {
+      if (!ignoreconfig.methods && !ignoreconfig.endpoints && !ignoreconfig.connections) {
```

Comment:
This condition is dangerous, because we will always forget to extend it.
Why do we even need this?
The conditions below already check the presence of the config key?